src/main/java/de/uni_passau/fim/se2/pipeline_helper/Main.java

```diff
@@ -113,10 +113,19 @@ public Integer call() throws Exception {
             validateParams();
 
             final Checker checker = buildChecker();
-            final CheckerResult result = checker.check();
-            CheckerResultWriter.writeFeedback(parent.outputDirectory, result);
+            CheckerResult result;
+            try {
+                result = checker.check();
+                System.out.println("Successfully produced a checker result.");
+            }
+            catch (Exception e) {
+                result = new CheckerResult(
+                    "Error", false,
+                    "The pipeline tool crashed. Please contact your instructor to resolve the issue."
```

Comment:
```suggestion
                    "The checker crashed. Please contact your instructor to resolve the issue."
```
nitpick: for students the steps are known as checkers.